test/Microsoft.AspNet.Razor.Runtime.Test/project.json

```diff
@@ -12,19 +12,27 @@
     },
     "Microsoft.AspNet.Testing": "1.0.0-*",
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "xunit.runner.aspnet": "2.0.0-aspnet-*"
+    "xunit": "2.1.0"
   },
+  "testRunner": "xunit",
   "commands": {
     "test": "xunit.runner.aspnet"
   },
+  "resource": "TestFiles/**",
```

Comment:
Does the `XmlDocumentationProvider` somehow read the XML file from a resource? If not, why make this change?